### PR TITLE
fixed the issue with the nav buttons remaining highlighted and the co…

### PR DESCRIPTION
**Pull request by Kovee98**
…ntact info appearing on refresh

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -40,7 +40,7 @@ html, body{
 	text-align: center;
 	transition: 300ms;
 }
-a.active, .navigation a:hover{
```
Comment (Kovee98) [inline css/style.css]: .active is a persistent attribute and is effective once it is clicked and won't go back to the original styling until the page is refreshed. In other words, you don't want it in here.

File: index.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h3>Junior Developer / Student</h3>
 	</div>
 	<div class="resume">
 		<p>
-			This is the resume page.<br>
+			<br>This is the resume page.
```
Comment (Kovee98) [inline index.html]: sorry, you can pretty much ignore these changes, I was trying to see why I couldn't see the words on this page. It's reporting that the div height is 0 pixels when I inspected the page in chrome.

File: css/style.css (hunk 1 above)
Comment (ethanwllms) [reply css/style.css]: Awesome! Thank you for pointing that out. That was something rather new to me.

File: index.html [hunk 3]
```diff
@@ -62,13 +62,7 @@ <h3>Junior Developer / Student</h3>
...
 
 	$(document).ready(function(){
-		$('a').click(function(){
-			//alert("i am click");
-			var selected = $(this);
-			$('a').removeClass('.active');
-			$(selected).addClass('active');
-		})
-
+
```
Comment (Kovee98) [inline index.html]: I couldn't find a purpose for this but you can add it back in if you had a purpose.

File: css/style.css (hunk 1 above)
Comment (Kovee98) [reply css/style.css]: No problem!

**Comment by ethanwllms** [comment]
Kovee98 Updates merged and reviewed. Thanks again @Kovee98

**Comment by Kovee98** [comment]
It was my pleasure! Please let me know if you need any extra help!

**Pull request merged**